db/migrate/20160921184937_add_cancel_reason_detail_to_replication_transfer.rb

```diff
@@ -0,0 +1,5 @@
+class AddCancelReasonDetailToReplicationTransfer < ActiveRecord::Migration
+  def change
+    add_column :replication_transfers, :cancel_reason_detail, :text, null: true
+  end
+end
```

Comment:
I don't know which is the default, but I prefer to be explicit.  What do you think about the limit?  There's likely a different default for each database adapter.